charts/exposurelog/templates/configmap-fs-mounts.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
That's correct, apiVersion here is for the kubernetes api.

Reply:
Is that different than the apiVersion in Chart.yaml (which is `v2`)? I thought that was also used as a Kubernetes version and was worried they would be required to match.

Reply:
Yes. `apiVersion` in the Chart.yaml is the Helm API version. The chart is not a Kubernetes API object. In general, Kubernetes objects might have different API groups and versions and they don't need to match. For example, a `ConfigMap` has `apiVersion: v1` a `Deployment` belongs to the `apps` API group and has `apiVersion: apps/v1`, etc